inst/templates/ad_adlb.R

```diff
@@ -380,6 +380,7 @@ adlb <- adlb %>%
 adlb <- adlb %>%
   # get MINIMUM value
   derive_extreme_records(
+    dataset_add = adlb,
     by_vars = exprs(STUDYID, USUBJID, PARAMCD, BASETYPE),
     order = exprs(AVAL, ADT, AVISITN),
     mode = "first",
```

Comment:
The comment below and `& AVISITN < 9997` from `filter_add` can be removed.

Reply:
Updated, thanks!